x/ccv/provider/keeper/keeper_test.go

```diff
@@ -197,3 +205,243 @@ func (suite *KeeperTestSuite) TestInitHeight() {
 		suite.Require().EqualValues(t.expected, height)
 	}
 }
+
+func (suite *KeeperTestSuite) TestHandleSlashPacketDistribution() {
```

Comment:
generally in favor of more scoped module table-driven tests

Reply:
Just curious, what do you mean by 'more scoped module'? (The table part I got).

Reply:
Sorry typo. Should just be `more scoped`, so more unit-testy is preferred